src/nvim/api/vim.c

```diff
@@ -385,7 +385,10 @@ Integer nvim_input(uint64_t channel_id, String keys)
 ///                 The same specifiers are used as for a key press, except
 ///                 that the "-" separator is optional, so "C-A-", "c-a"
 ///                 and "CA" can all be used to specify Ctrl+Alt+click.
-/// @param grid Grid number if the client uses |ui-multigrid|, else 0.
+/// @param grid Grid number if the client uses |ui-multigrid|, with manual positioning of the
+///             windows else 0. If you are using multigrid and let nvim determine the positioning
+///             of the windows when using multigrid, you can pass 0 and nvim will find the window
+///             grid for you. You can combine both approaches.
```

Comment:
this is jumbled. don't need to over-explain. 0 = "nvim decides".

```suggestion
/// @param grid Grid number (used by |ui-multigrid| client), or 0 to let Nvim determine positioning of windows.
```

Reply:
I'm going to move the more detailed explanation to `:help dev-ui`, and link it from here. I think it needs to be clear that it's not OK to pass 0, when the UI takes over the window positioning and don't respect `compindex`, `screen_row` and `screen_col` (like all current UIs out there, except the development version of Neovide), as was noted by your previous comment on the same help text.

Reply:
More detailed documentation has been added to `develop.txt`. And this documentation simplified.